docs/topics/control-flow.md

```diff
@@ -6,30 +6,57 @@ In Kotlin, `if` is an expression: it returns a value.
 Therefore, there is no ternary operator (`condition ? then : else`) because ordinary `if` works fine in this role.
 
 ```kotlin
-var max = a 
-if (a < b) max = b
+val tax: Int
+val value = 50
+
+if (value < 100) tax = 10
 
 // With else 
-var max: Int
-if (a > b) {
-    max = a
+val tax: Int
+val value = 50
+
+if (value < 10) {
+    tax = 1
 } else {
-    max = b
+    tax = 10
 }
  
 // As expression 
-val max = if (a > b) a else b
+val tax = if (value < 10) 1 else 10
+
+// With else if
+val tax: Int
+val value = 50
+
+if (value < 10) {
+  tax = 1
+} else if (value < 100) {
+  tax = 10
+} else if (value < 1000) {
```

Comment:
I would remove this branch; I think it's enough to show one "else if" branch. Otherwise, this new example is very wordy and this additional branch doesn't add any valuable sense.